internal/scheduler/submitcheck.go

```diff
@@ -78,7 +78,7 @@ func NewSubmitChecker(
 
 func (srv *SubmitChecker) Run(ctx context.Context) error {
 	srv.updateExecutors(ctx)
-	ticker := time.NewTicker(1 * time.Minute)
```

Comment:
This seems to be a big change?

Reply:
Yeah, this will be made into a flag that is controlled from config :))